### DISCUSSION TITLE
Question: how to get a message in message_ready state that wouldn't be enqueued ?

### DISCUSSION THREAD

**Discussion by jeven2016** [michaelklishin/rabbit-hole#250] [General]
Hi  team,
   Do you have any ways to get a message in message_ready state that wouldn't be enqueued in abnormal situation?

Thanks.

**Comment by michaelklishin** [comment]
If you want to consume a message such that it won’t be automatically re-queued in case of a consumer failure, just use a consumer with the  [automatic acknowledgment mode](https://rabbitmq.com/confirms.html#acknowledgement-modes).

This is a feature provide by a [messaging client](https://github.com/rabbitmq/amqp091-go), not an HTTP API client like this one.
